lib/beacon/beacon.ex

```diff
@@ -194,8 +194,6 @@ defmodule Beacon do
 
     #{context}
 
-    Make sure you have created a page for this path.
```

Comment:
Removed because this function is reused not only for call inside pages so this phrase may get out of context